.github/workflows/main.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu, macOS, Windows]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-beta.3"]
```

Comment:
I'd like to use `3.11-dev` as our version here unless there's a compelling reason to pin to one beta version -- a moving target is arguably a better smoke test for us than a pinned release.